src/components/navbar/Navbar.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
 
-import { Ul, Li, Nav, A } from './NavbarStyled';
+import { Ul, Li, Nav, A, Img, Div, LogoLink, SignUpLink } from './NavbarStyled';
```

Comment:
I understand why you named these components this way, but it's common practice to name them a bit more descriptively. For example, `Ul` could be `NavItems` and `Li` be `NavItem`